src/main/java/com/prgrms/vouchermanager/domain/customer/Customer.java

```diff
@@ -0,0 +1,35 @@
+package com.prgrms.vouchermanager.domain.customer;
+
+import lombok.Getter;
+
+import java.util.UUID;
+
+@Getter
+public class Customer {
+    private final UUID id;
+    private final String name;
+    private final int yearOfBirth;
+    private final boolean isBlacklist;
+    public Customer(UUID id, String name, int yearOfBirth, boolean isBlacklist) {
+        this.id = id;
+        this.name = name;
+        this.yearOfBirth = yearOfBirth;
+        this.isBlacklist = isBlacklist;
+    }
+
+    public Customer(String name, int yearOfBirth) {
+        this.id = UUID.randomUUID();
+        this.name = name;
+        this.yearOfBirth = yearOfBirth;
+        this.isBlacklist = false;
+    }
```

Comment:
```suggestion
    public Customer(String name, int yearOfBirth) {
        this(UUID.randomUUID(), name, yearOfBirth, false);
    }

    public Customer(UUID id, String name, int yearOfBirth, boolean isBlacklist) {
        this.id = id;
        this.name = name;
        this.yearOfBirth = yearOfBirth;
        this.isBlacklist = isBlacklist;
    }
```

오버로딩 사용하여 객체 생성 시에 초기화 하는 부분을 통일 시키는 것이 좋습니다.